### ISSUE TITLE
"ebs_csi" policy documents will fail when "ebs_csi_kms_cmk_ids" is set

### ISSUE THREAD

**Issue by ahvandf** [terraform-aws-modules/terraform-aws-iam#211]
## Description

Please provide a clear and concise description of the issue you are encountering, your current setup, and what steps led up to the issue. If you can provide a reproduction, that will help tremendously.

in `aws_iam_policy_document.ebs_csi` dynamic block, `for_each` expression is defined as below
`for_each = length(var.ebs_csi_kms_cmk_ids) > 0 ? [1] : []`
and **resources** which should be a set of strings is defined as 
`resources = statement.value` 
which cause the module to fail with **Incorrect attribute value type** error if **ebs_csi_kms_cmk_ids** input is set

```
 Error: Incorrect attribute value type
 
   on .terraform/modules/ebs_csi_irsa/modules/iam-role-for-service-accounts-eks/policies.tf line 259, in data "aws_iam_policy_document" "ebs_csi":
  259:       resources = statement.value
 
 Inappropriate value for attribute "resources": set of string required.


 Error: Incorrect attribute value type
 
   on .terraform/modules/ebs_csi_irsa/modules/iam-role-for-service-accounts-eks/policies.tf line 280, in data "aws_iam_policy_document" "ebs_csi":
  280:       resources = statement.value
 
 Inappropriate value for attribute "resources": set of string required.
```

## ⚠️ Note

Before you submit an issue, please perform the following first:

1. Remove the local `.terraform` directory (! ONLY if state is stored remotely, which hopefully you are following that best practice!): `rm -rf .terraform/`
2. Re-initialize the project root to pull down modules: `terraform init`
3. Re-attempt your terraform plan or apply and check if the issue still persists

## Versions

 - Terraform: 1.1.7

 - Provider(s):
kubectl v1.13.1
aws v4.5.0
cloudinit v2.2.0
helm v2.4.1
kubernetes v2.8.0
local v2.2.2
null v3.1.1
random v3.1.2
template v2.2.0
tls v3.1.0
http v2.4.1

 - Module:
iam-role-for-service-accounts-eks

## Reproduction

set "ebs_csi_kms_cmk_ids" and execute the module

## Expected behavior

for_each expression should take the kms key as input and **resources** element should be a set of string

**Comment by bryantbiggs** [comment]
I believe this would be resolved with https://github.com/terraform-aws-modules/terraform-aws-iam/pull/203

**Comment by bryantbiggs** [comment]
resolved in #203
